src/OpenTelemetry/README.md

```diff
@@ -5,14 +5,16 @@
 
 * [Installation](#installation)
 * [Introduction](#introduction)
-* [Basic usage](#basic-usage)
-* [Advanced usage scenarios](#advanced-usage-scenarios)
-  * [Customize Exporter](#customize-exporter)
+* [Getting started](#getting-started)
+* [Customization](#customization)
   * [Customize Sampler](#customize-sampler)
   * [Customize Resource](#customize-resource)
   * [Filtering and enriching activities using
     Processor](#filtering-and-enriching-activities-using-processor)
   * [OpenTelemetry Instrumentation](#opentelemetry-instrumentation)
+* [Advanced topics](#advanced-topics)
+  * [Build your own Exporter](#build-your-own-exporter)

```

Comment:
I'm on the fence here - for consistency this should not start with a verb (e.g. `Build`), instead it should be `Building`. And `Customize Sampler` should be `Customizing Sampler`.

Not a native English speaker so I need some suggestion from professionals, happy to give my _authentic Chinese translation_ if we need it later 😄

Reply:
I asked google translate... 建造. How'd I do? Hardly a professional here 😆.

I do like `Building` vs `Build`. Other option may be `Implementing an Exporter`.

Also instead of `Customization` maybe `Configuration`?

Reply:
Thanks! I will take your words 👍

Reply:
> I do like `Building` vs `Build`. Other option may be `Implementing an Exporter`.

I'll use `Building` for now. Personally I would try to avoid running into `Implementing an Exporter` vs. `Implementing Exporters` (look at the names we have in this project, `trace` is singular form, `metrics` are plural form, and we have `Benchmarks` in plural form under a folder called `test` which is singular form, and the test projects all have `Tests` in their name 🧙).

Reply:
Ha. Yes all good points. Words are hard. `Building` is good in this context.

Perhaps `Trace` is better as `Tracing` in the context of a namespace. Even the specs name it the `Tracing API` and have the doc in a folder called `trace` https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md. .NET has a similar namespace [System.Diagnostics.Tracing](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.tracing)

Reply:
Take a look at [this](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/library-layout.md#api-package).
It was lots of people debating over months if not years over these terms, and all of them had good intentions 😄

Reply:
Wow thanks for sharing. Excellent context. This kind of collaboration is always difficult - especially across languages!

Reply:
I always tell people naming things is the hardest part of writing software! Fair to say `metrics` are to `traces` as `measuring` is to `tracing`?